src/actions/auth/signin.js

```diff
@@ -0,0 +1,32 @@
+import jwt from 'jsonwebtoken';
+import { ACTIONS } from '../actionTypes';
+import instance, { setToken } from '../../config/http';
+
+export const setCurrent = user => ({
```

Comment:
Hello @pitaz, please rename this const to reflect what it is doing e.g setCurrentUser

Reply:
OK. Thanks